tests/testthat/test-climatic_summary.R

```diff
@@ -0,0 +1,42 @@
+library(RInstatClimatic)
+x_daily <- daily_niger %>% dplyr::group_by(date) %>% dplyr::summarize(mean_rain = mean(rain), st_dv_rain = sd(rain), n_na_rain = naflex::na_n(rain))
+x_monthly <- daily_niger %>% dplyr::group_by(month) %>% dplyr::summarize(mean_rain = mean(rain), st_dv_rain = sd(rain), n_na_rain = naflex::na_n(rain))
+x_annual <- daily_niger %>% dplyr::group_by(year) %>% dplyr::summarize(mean_rain = mean(rain), st_dv_rain = sd(rain), n_na_rain = naflex::na_n(rain))
```

Comment:
I think these should be grouped by station as well to match the expected outputs?